Dockerfile.ci

```diff
@@ -1,16 +1,24 @@
 ARG image_tag=latest
+FROM elifesciences/proofreader-php:0.2 AS proofreader
 FROM elifesciences/journal_composer_dev:${image_tag} AS composer
 FROM elifesciences/journal_cli:${image_tag}
 
 USER root
 RUN mkdir -p app build/ci && \
-    chown www-data:www-data build/ci
+    touch .php_cs.cache && \
+    chown --recursive www-data:www-data build .php_cs.cache
+
+COPY --from=proofreader --chown=elife:elife /srv/proofreader-php /srv/proofreader-php
+RUN ln -s /srv/proofreader-php/bin/proofreader /srv/bin/proofreader
 
 COPY --chown=elife:elife \
     behat.yml.dist \
     phpunit.xml.dist \
     .php_cs \
     ./
+COPY --chown=elife:elife .ci/ .ci/
+COPY --chown=elife:elife web/ web/
+COPY --chown=elife:elife composer.lock .
```

Comment:
The security check could technically be run on the dev Composer image (single command, so not through a script).